rdkit/Chem/ChemUtils/DescriptorUtilities.py

```diff
@@ -21,3 +21,35 @@ def wrapper(func):
     func.version = version
     return func
   return wrapper
+
+class VectorDescriptorWrapper:
+    """Wrap a function that returns a vector and make it seem like there
+    is one function for each entry.  These functions are added to the global
+    namespace with the names provided"""
+    def __init__(self, func, names, version, namespace):
+        self.func = func
+        self.names = names
+        self.func_key = "__%s"%(func.__name__)
+        for i,n in enumerate(names):
+            def f(mol, key=self._get_key(i)):
+                return self.call_desc(mol, key)
+            f.__name__ = n
+            f.__qualname__ = n
+            f.version = version
+            namespace[n] = f
+
+    def _get_key(self, index):
+        return "%s%s"%(self.func_key, index)
+    
+    def call_desc(self, mol, key):
```

Comment:
I think it’s really useful to have a force argument that allows you to recalculate the property even if it’s already there